pyproject.toml

```diff
@@ -67,8 +67,8 @@ dev = [
 test = ["pytest", "scipy"]
 marine = ["marine>=0.0.5"]
 
-[tool.setuptools.packages]
-find.namespaces = false
+[tool.setuptools.packages.find]
+include = ["pyopenjtalk*"]
```

Comment:
Get the warning: `Package would be ignored`.
The cause is that you set `namespaces = false`.

`pyoenjtalk/htsvoice` should have been a `namespace package`.
https://setuptools.pypa.io/en/latest/userguide/datafiles.html#subdirectory-for-data-files
https://setuptools.pypa.io/en/latest/userguide/package_discovery.html#finding-simple-packages